tests/test_wave_05.py

```diff
@@ -52,13 +52,19 @@ def test_new_genre_rec_from_empty_friends():
             }
         ]
     }
+    #Act
+    recommendations = get_new_rec_by_genre(sonyas_data)
+
+    #Assert
+    assert len(recommendations) == 0
+    assert not recommendations
+    assert recommendations == []
```

Comment:
I'd only do one of these checks. assert recommendations == [] checks that the result is an empty list, while assert len(recommendations) == 0 checks only that recommendations is something reporting a length of 0.

If you really want to be sure that the return value is truly a list, then go with `== []`, but I'd tend towards the latter (we tend to be a little loose with types in Python).